packages/bun-uws/src/LoopData.h

```diff
@@ -44,7 +44,11 @@ struct alignas(16) LoopData {
 
     /* Map from void ptr to handler */
     std::map<void *, MoveOnlyFunction<void(Loop *)>> postHandlers, preHandlers;
-
+    /* Cork data */
+    char *corkBuffer = new char[CORK_BUFFER_SIZE];
```

Comment:
We would need to free this

If we instead make this statically allocated, we wouldn't need to worry about freeing it:
```suggestion
    char corkBuffer[CORK_BUFFER_SIZE];
```

Reply:
oh this already existed